src/components/tools/listing/TableView.js

```diff
@@ -0,0 +1,251 @@
+import ids from "../ids";
+import React from "react";
+import {
+    build,
+    DECheckbox,
+    EmptyTable,
+    EnhancedTableHead,
+    formatMessage,
+    getMessage,
+    withI18N,
+} from "@cyverse-de/ui-lib";
+import {
+    Paper,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableRow,
+} from "@material-ui/core";
+import { injectIntl } from "react-intl";
+import messages from "../Messages";
+import TableLoading from "../../utils/TableLoading";
+
+const buildId = build;
+
+/**
+ * Returns localized column header information for the tool listing table.
+ * @param {Object} intl - the internationalization object
+ */
+const columnData = (intl) => [
+    {
+        id: ids.NAME,
+        name: formatMessage(intl, "name"),
+        numeric: false,
```

Comment:
I believe `numeric` has been replaced with `align`, so this might be come `align: "left"` instead, for example

Reply:
It looks like explicitly specifying the `numeric` property to `false` isn't necessary, so I'll simply remove these. I did see `numeric` listed in the properties for `EnhancedTableHead`; it appears that it's used to set the `align` property. I don't know if that's the preferred way to set the alignment, though. Should I avoid using this property in the future?

Reply:
Ah, no, you're correct - for some reason I remembered it being removed entirely, but it's still supported.  `align` makes more sense to me, but either is fine in that case.